content/en/docs/concepts/storage/volumes.md

```diff
@@ -1376,119 +1344,118 @@ persistent volume:
 
-Vendors with external CSI drivers can implement raw block volumes support
+Vendors with external CSI drivers can implement raw block volume support
 in Kubernetes workloads.
 
-You can [setup your PV/PVC with raw block volume support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support)
-as usual, without any CSI specific changes.
+You can set up your
+[PersistentVolume/PersistentVolumeClaim with raw block volume support](/docs/concepts/storage/persistent-volumes/#raw-block-volume-support) as usual, without any CSI specific changes.
 
 #### CSI ephemeral volumes
 
 {{< feature-state for_k8s_version="v1.16" state="beta" >}}
 
 You can directly configure CSI volumes within the Pod
 specification. Volumes specified in this way are ephemeral and do not
-persist across Pod restarts. See [Ephemeral
+persist across pod restarts. See [Ephemeral
 Volumes](/docs/concepts/storage/ephemeral-volumes/#csi-ephemeral-volume)
 for more information.
 
-#### {{% heading "whatsnext" %}}
-
-For more information on how to develop a CSI driver, refer to the [kubernetes-csi
-documentation](https://kubernetes-csi.github.io/docs/)
+For more information on how to develop a CSI driver, refer to the
+[kubernetes-csi documentation](https://kubernetes-csi.github.io/docs/)
 
 #### Migrating to CSI drivers from in-tree plugins
 
 {{< feature-state for_k8s_version="v1.14" state="alpha" >}}
 
-The CSI Migration feature, when enabled, directs operations against existing in-tree
+The `CSIMigration` feature, when enabled, directs operations against existing in-tree
 plugins to corresponding CSI plugins (which are expected to be installed and configured).
 The feature implements the necessary translation logic and shims to re-route the
 operations in a seamless fashion. As a result, operators do not have to make any
 configuration changes to existing Storage Classes, PVs or PVCs (referring to
 in-tree plugins) when transitioning to a CSI driver that supersedes an in-tree plugin.
 
-In the alpha state, the operations and features that are supported include
+The operations and features that are supported include:
 provisioning/delete, attach/detach, mount/unmount and resizing of volumes.
 
-In-tree plugins that support CSI Migration and have a corresponding CSI driver implemented
-are listed in the "Types of Volumes" section above.
+In-tree plugins that support `CSIMigration` and have a corresponding CSI driver implemented
+are listed in [Types of Volumes](#volume-types).
 
 ### FlexVolume {#flexVolume}
 
-FlexVolume is an out-of-tree plugin interface that has existed in Kubernetes
+`FlexVolume` is an out-of-tree plugin interface that has existed in Kubernetes
 since version 1.2 (before CSI). It uses an exec-based model to interface with
-drivers. FlexVolume driver binaries must be installed in a pre-defined volume
-plugin path on each node (and in some cases master).
+drivers. The `FlexVolume` driver binaries must be installed in a pre-defined volume
```

Comment:
```suggestion
drivers. The FlexVolume driver binaries must be installed in a pre-defined volume
```

Reply:
Not sure about this one. I consider a driver name similar to a namespace name, or plugin name, or library name.